executor/cook/progress.py

```diff
@@ -142,8 +142,12 @@ def tail(self, sleep_time_ms):
                 logging.debug('{} has not yet been created, sleeping {} ms'.format(self.target_file, sleep_time_ms))
                 time.sleep(sleep_param)
 
+            if not os.path.isfile(self.target_file):
```

Comment:
Are we expecting the task to create this file?  Should we consider having the executor create it?

Reply:
I think the task should be responsible for creating the file. E.g. when the file is `stdout` on Mesos 0.27, the executor doesn't have enough privileges to create it.